website/docusaurus.config.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
```

Comment:
This site is now v2 so this comment is inaccurate. Let's remove the entire comment.

Reply:
Done.